src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemExtensionsTests.cs

```diff
@@ -271,6 +271,49 @@ public void SarifWorkItemExtensions_CreateWorkItemDescription_MultipleResults()
             description.Should().BeEquivalentTo($"This work item contains {numOfResult} '{toolName}' issue(s) detected in {string.Format(firstLocation, 1)} (+{additionLocationCount} locations). Click the 'Scans' tab to review results.");
         }
 
+        [Fact]
+        public void SarifWorkItemExtensions_CreateWorkItemDescription_ResultsShouldNotBeFiled()
+        {
+            string toolName = "TestToolName";
+            string firstLocation = @"C:\Test\Data\File{0}sarif";
```

Comment:
>@"C:\Test\Data\File{0}sarif" [](http://example.com/codeflow?start=35&length=28)

suggest change to not using windows specific format, like:
Path.Combine(Directory.GetCurrentDirectory(), "Test", "Data", "File{0}sarif");